support-frontend/assets/pages/subscriptions-redemption/components/productSummary/productSummaryStyles.js

```diff
@@ -62,8 +66,8 @@ export const textBlock = css`
   }
 
   h3 {
-    ${body.medium({ fontWeight: 'bold' })};
-    margin: -5px 0 -3px;
+    ${headline.xxsmall({ fontWeight: 'bold' })};
+    margin: 8px 0 2px;
```

Comment:
It would be great to use space here if possible: 
```margin: ${space[2]}px 0 ${space[1]}px;```

Reply:
2px is actually half a space though! But I've changed to use space wherever possible.

Reply:
Yeah, fair enough 😄